knowledge-base/visual-studio-preview-is-not-working-when-using-objectdatasource.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Report Designer Preview is not working when using ObjectDataSource
+description: Preview the report in the Visual Studio Designer is not working when using ObjectDataSource.
+type: troubleshooting
+page_title: Reports using ObjectDataSource cannot be opened with the Visual Studio Designer
+slug: visual-studio-preview-is-not-working-when-using-objectdatasource
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Reporting</td>
+	</tr>
+    <tr>
+		<td>Report Designer</td>
+		<td>Visual Studio Designer</td>
+	</tr>
+</table>
+
+## Description
+
+Generally, the Visual Studio Report Designer operates *inside* the Visual Studio application. Thus, when previewing a report the report constructor is executed in the context of the Visual Studio application and all the configuration settings are obtained from the Visual Studio's **devenv.exe.config** file. If not taken upon consideration, this might lead to two major issues - *ConnectionStrings* and *AssemblyReferences* mismatch.   
+
+## Solution
+
+Make the following are correctly set up:
+
+- Connection Strings
+
+    As described in MSDN, the [ConfigurationManager.ConnectionStrings property](https://docs.microsoft.com/en-us/dotnet/api/system.configuration.configurationmanager.connectionstrings?redirectedfrom=MSDN&view=net-5.0#System_Configuration_ConfigurationManager_ConnectionStrings) gets the *ConnectionStringsSection* data for the current application's default configuration. Generally in case of Visual Studio this is devenv.exe.config. The [SqlDataSource component](../sqldatasource) is particularly designed to use the current's project configuration file. However, if your custom data access layer (DAL) is reading the **ConfigurationManager**.ConnectionStrings from the report that is previewed you get all connection strings defined in the devenv.exe.config file. In order to avoid the error you have several options:  
```

Comment:
Avoid using "Generally" - remove it from this paragraph, like this: "... gets the *ConnectionStringsSection* data for the current application's default configuration - in case of Visual Studio this is devenv.exe.config."